libstorage/MemoryTable2.cpp

```diff
@@ -126,8 +127,8 @@ Entries::Ptr MemoryTable2::selectNoLock(const std::string& key, Condition::Ptr c
     }
     catch (std::exception& e)
     {
-        STORAGE_LOG(ERROR) << LOG_BADGE("MemoryTable2") << LOG_DESC("Table select failed for")
-                           << LOG_KV("msg", boost::diagnostic_information(e));
+        STORAGE_LOG(ERROR) << LOG_BADGE("MemoryTable2") << LOG_DESC("Table select failed")
+                           << LOG_KV("what", e.what());
```

Comment:
The same as  before, why use `e.what` either than `boost::diagnostic_information`

Reply:
because it is useless, output the same as e.what(). the difference is diagnostic make log untidy